src/test/features/offer/form/models/offer.ts

```diff
@@ -69,10 +69,10 @@ describe('Offer', () => {
 
     describe('should accept when', () => {
       it('offer text and future date', () => {
-        const futureDate = moment().add(10, 'days')
-        const date = new LocalDate(futureDate.year(), futureDate.month(), futureDate.day())
-        const errors = validator.validateSync(new Offer('offer text', date))
-        expect(errors.length).to.equal(1)
+        const futureDate = moment().add(10, 'month')
```

Comment:
Why did you have to make that change? It looks not related, smells like something bad is happening here.